packages/template-chakra-storefront/src/pages/product-list/partials/empty-results.jsx

```diff
@@ -9,18 +9,43 @@ import React, {Fragment} from 'react'
 import {Button, Text, Flex, Stack, Link} from '@chakra-ui/react'
 import PropTypes from 'prop-types'
 import {Link as RouteLink} from 'react-router-dom'
-import {defineMessage, FormattedMessage, useIntl} from 'react-intl'
 import {SearchIcon} from '../../../components/icons'
 import RecommendedProducts from '../../../components/recommended-products'
 import {EINSTEIN_RECOMMENDERS} from '../../../../config/constants'
+import {withTranslations, defineMessages} from '../../../components/with-translations'
 
-const contactUsMessage = defineMessage({
-    id: 'empty_search_results.link.contact_us',
-    defaultMessage: 'Contact Us'
+const messages = defineMessages({
```

Comment:
I like how the messages are collected in this block... set aside for readability and yet still on the same file for easy searching.